Formula/mariadb-connector-odbc.rb

```diff
@@ -0,0 +1,66 @@
+class MariadbConnectorOdbc < Formula
+  desc "MariaDB Standardized database driver"
+  homepage "https://downloads.mariadb.org/connector-odbc/"
+  # MariaDB's download page doesn't let you download without going through the site, probably a cookie-based check
+  # The connector is packaged at http://archive.mariadb.org/connector-odbc-2.0.13/source/mariadb-connector-odbc-2.0.13-ga-src.tar.gz, but there's no https here
+  # So the next best choice seems to be github:
+  url "https://github.com/MariaDB/mariadb-connector-odbc/archive/2.0.13.tar.gz"
+  sha256 "6b77e879335ea32dad40d42de95930df8ad557faa887b414a09ab7ead3ce963c"
+
+  depends_on "cmake" => :build
+  depends_on "mariadb-connector-c"
+  depends_on "unixodbc"
+  depends_on "openssl"
+
+  patch :DATA
```

Comment:
Great. Can you remove the `patch :DATA` and use the [upstream commit URL](https://github.com/MariaDB/mariadb-connector-odbc/commit/677f3c9d2ddb2100d113289f0314520c46ef5680.diff) in a `patch do` block?